src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/Microsoft.WinFX.targets

```diff
@@ -397,6 +404,59 @@
 
   </Target>
 
+  <!--
+                ==========================================
+                  CreateTemporaryTargetAssemblyProject
+                ==========================================
+                Name : CreateTemporaryTargetAssemblyProject
+  -->
+
+   <Target Name="CreateTemporaryTargetAssemblyProject"
+               Condition="'$(ShouldWPFSupportPackageReferences)' == 'true' and '$(_RequireMCPass2ForMainAssembly)' == 'true'" >
+
+       <CreateTemporaryTargetAssemblyProject 
+                CurrentProject="$(MSBuildProjectFullPath)"
+                GeneratedCodeFiles="@(_GeneratedCodeFiles)"
+                CompileTypeName="Compile"
+                ReferencePath="@(ReferencePath)"
+                ReferencePathTypeName="ReferencePath" 
+                IntermediateOutputPath="$(IntermediateOutputPath)"
+                AssemblyName="$(AssemblyName)" 
+                MSBuildProjectExtensionsPath="$(MSBuildProjectExtensionsPath)" 
+                Analyzers="$(Analyzers)" >
+           <Output TaskParameter="TemporaryTargetAssemblyProjectName" PropertyName="_TemporaryTargetAssemblyProjectName"/>
+       </CreateTemporaryTargetAssemblyProject>
+
+      <Message Text="GenerateTemporaryTargetAssembyProject project is $(_TemporaryTargetAssemblyProjectName)" />
+
+      <!-- Path to generated NuGet props and targets files -->
+      <Message Text="MSBuildProjectExtensionsPath: $(MSBuildProjectExtensionsPath)" />
+
+      <!-- Get the name of the outer WPF parent project -->
+      <PropertyGroup>
+          <CurrentProjectName>$([System.IO.Path]::GetFileName('$(MSBuildProjectFullPath)'))</CurrentProjectName>
+      </PropertyGroup>
+
+      <!-- Collect the generated NuGet files from the parent project required support PackageReferences -->
+      <ItemGroup>
+        <SourceGeneratedNuGetPropsAndTargets Include="$(MSBuildProjectExtensionsPath)\*.props"/>
```

Comment:
Should we only include the `*.nuget.g.props` files?

Reply:
Good question - why would this not be limited to nuget specific props here?

Reply:
No, it should only take the *.nuget.g.* files which should include the correct references to any other props/targets files for msbuild extensions.   

```xml
   <SourceGeneratedNuGetPropsAndTargets Include="$(MSBuildProjectExtensionsPath)\*.nuget.g.props"/>
   <SourceGeneratedNuGetPropsAndTargets Include="$(MSBuildProjectExtensionsPath)\*.nuget.g.targets"/>
```

```
<name>.csproj.nuget.dgspec.json
<name>.csproj.nuget.g.props
<name>.csproj.nuget.g.targets
```

It should only be picking up these generated files.

Reply:
(I've made the update.)